File: final-project/src/main/java/com/example/final_project/dto/AddressDTO.java
package com.example.final_project.dto;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class AddressDTO {
    private String province;
    private String provinceCode;

    private String district;
    private String districtCode;

    private String ward;
    private String wardCode;

    private String addressDetail;
}

File: lib/models/address_dto.dart
class AddressDTO {
  final String province;
  final String provinceCode;
  final String district;
  final String districtCode;

  AddressDTO({
    required this.province,
    required this.provinceCode,
    required this.district,
    required this.districtCode, required String specificAddress, required wardCode, required ward,
  });

  Map<String, dynamic> toJson() => {
    'province': province,
    'provinceCode': provinceCode,
    'district': district,
    'districtCode': districtCode,
  };
}
